packages/reports/src/Stores/Modules/Profit/Helpers/__tests__/format-response.spec.ts

```diff
@@ -0,0 +1,99 @@
+    beforeEach(() => {
+        mockProfitTableTransactionData = {
+            app_id: 11780,
+            buy_price: 1,
+            contract_id: 55113405821,
+            contract_type: 'CALL',
+            duration_type: 'minutes',
+            longcode:
+                'Win payout if Volatility 100 (1s) Index is strictly higher than entry spot at 15 minutes after contract start time.',
+            payout: 1.96,
+            purchase_time: 1700539661,
+            sell_price: 0.61,
+            sell_time: 1700539816,
+            shortcode: 'CALL_1HZ100V_1.96_1700539661_1700540561_S0P_0',
+            transaction_id: 111189668101,
+            underlying_symbol: '1HZ100V',
+        };
+    });
+
+    it('should return correct formatted object', () => {
+        const currency = 'USD';
+        const active_symbols: ActiveSymbols = [];
+        const returnValue = formatProfitTableTransactions(mockProfitTableTransactionData, currency, active_symbols);
+        expect(returnValue).toEqual({
+            app_id: 11780,
+            buy_price: 1,
+            contract_id: 55113405821,
+            contract_type: 'CALL',
+            display_name: '',
+            duration_type: 'minutes',
+            longcode:
+                'Win payout if Volatility 100 (1s) Index is strictly higher than entry spot at 15 minutes after contract start time.',
+            payout: 1.96,
+            purchase_time: '21 Nov 2023 04:07:41',
+            purchase_time_unix: 1700539661,
+            sell_price: 0.61,
+            sell_time: '21 Nov 2023 04:10:16',
+            shortcode: 'CALL_1HZ100V_1.96_1700539661_1700540561_S0P_0',
+            transaction_id: 111189668101,
+            underlying_symbol: '1HZ100V',
+            profit_loss: '-0.39',
+        });
+    });
+
+    it('should not return purchase time if purchase time is not available', () => {
+        mockProfitTableTransactionData.purchase_time = undefined;
+        const currency = 'USD';
+        const active_symbols: ActiveSymbols = [];
+        const returnValue = formatProfitTableTransactions(mockProfitTableTransactionData, currency, active_symbols);
+        expect(returnValue.purchase_time).toBeUndefined();
+    });
+
+    it('should return profit loss for with correct BTC formatted value', () => {
+        mockProfitTableTransactionData.buy_price = 0.0001;
+        mockProfitTableTransactionData.sell_price = 0.0002;
+        const currency = 'BTC';
+        const active_symbols: ActiveSymbols = [];
+        const returnValue = formatProfitTableTransactions(mockProfitTableTransactionData, currency, active_symbols);
+        expect(returnValue.profit_loss).toEqual('0.00010000');
+    });
+
+    it('should return positive profit loss if sell price is more than buy price', () => {
+        mockProfitTableTransactionData.sell_price = 1;
+        mockProfitTableTransactionData.buy_price = 0.5;
+        const currency = 'USD';
+        const active_symbols: ActiveSymbols = [];
```

Comment:
In 3 cases variables are the same (`const active_symbols: ActiveSymbols = []; `) ; we can create a constant above these cases and reuse it. Smth like:
`const emptyActiveSymbols: ActiveSymbols = [];`
Applicable to `currency` as well.
Also, if possible, please use `camalCase`